content/en/docs/tasks/policy-enforcement/rate-limit/index.md

```diff
@@ -0,0 +1,333 @@
+individual `productpage` instance that will allow 10 requests per minute. In this way, we will ensure that our `productpage`
+service handles a maximum of 1 requests per minute through instance gateway, but each `productpage` instance can handle
+up to 10 requests per minute allowing for any in-mesh traffic.
+
+## Before you begin
+
+1. Setup Istio in a Kubernetes cluster by following the instructions in the
+   [Installation Guide](/docs/setup/getting-started/).
+
+1. Deploy the [Bookinfo](/docs/examples/bookinfo/) sample application.
+
+## Rate limits
+
+Envoy supports two kinds of rate limiting: global and local. Global rate
+limiting uses a global gRPC rate limiting service to provide rate limiting for the entire mesh.
+Local rate limiting is used to limit the rate of requests per service instance.
+Local rate limiting can be used in conjunction with global rate limiting to reduce load on
+the global rate limiting service.
+
+In this task you will configure Envoy to rate limit traffic to a specific path of a service
+using both global and local rate limits.
+
+### Global rate limit
+
+Envoy can be used to [set up global rate limits](https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/other_features/global_rate_limiting) for your mesh.
+
+1. Global rate limiting in Envoy uses a gRPC API for requesting quota from a rate limiting service.
+   A [reference implementation](https://github.com/envoyproxy/ratelimit) of the API, written in Go with a Redis
+   backend, is used below.
+
+   The following [configmap](https://github.com/envoyproxy/ratelimit#configuration) configures the
+   reference implementation to rate limit requests to the path `/productpage` at 1 req/min and all other
+   requests at 100 req/min.
+
+   {{<text yaml >}}
+   apiVersion: v1
+   kind: ConfigMap
+   metadata:
+     name: ratelimit-config
+   data:
+     config.yaml: |
+       domain: productpage-ratelimit
+       descriptors:
+         - key: PATH
+           value: "/productpage"
+           rate_limit:
+             unit: minute
+             requests_per_unit: 1
+         - key: PATH
+           rate_limit:
+             unit: minute
+             requests_per_unit: 100
+   {{< /text >}}
+
+1. Apply an `EnvoyFilter` to the `ingressgateway` to enable global rate limiting using Envoy's global rate limit filter.
+
+   The first patch inserts the
+   `envoy.filters.http.ratelimit` [global envoy filter](https://www.envoyproxy.io/docs/envoy/latest/api-v3/extensions/filters/http/ratelimit/v3/rate_limit.proto#envoy-v3-api-msg-extensions-filters-http-ratelimit-v3-ratelimit) filter into the `HTTP_FILTER` chain.
+   The `rate_limit_service` field specifies the external rate limit service, `rate_limit_cluster` in this case.
+
+   The second patch defines the `rate_limit_cluster`, which provides the endpoint location of the external rate limit service.
+
+   {{< text bash >}}
```

Comment:
This code block must be indented 4 spaces (see lint error)

Reply:
fixed